README.md

```diff
@@ -0,0 +1,92 @@
+# spark8t toolkit
```

Comment:
please add a header mentioning that we are hiring and folks should apply at https://canonical.com/careers